### DISCUSSION TITLE
Hair covers hat.

### DISCUSSION THREAD

**Discussion by Mudkipp** [Luke100000/minecraft-comes-alive#398] [Q&A]
In the past version, hats were prioritized over hair, but now hats are covered by hair. What should I do? And there's still Steve Skin.
![2022-09-15_13 01 34](https://user-images.githubusercontent.com/113573319/190311978-0fcd8714-29ea-4e1e-9aaa-048e0873e3be.png)
![2022-09-15_13 01 58](https://user-images.githubusercontent.com/113573319/190312011-7810b192-b6cf-497e-9241-e71a71cc053b.png)

**Comment by CDAGaming** [comment]
The steve skin has not been present when we tried to reproduce that, though I'll CC @Luke100000 to try again to reproduce it (Though in any case, its unlikely that can even be fixed, unless something was missed)

Also, which past MCA version do you refer to, as there have been a few since your last posting @Mudkipp
